node/src/db_config/persistent_configuration.rs

```diff
@@ -571,11 +571,9 @@ impl PersistentConfigurationReal {
         value: Option<T>,
```

Comment:
I know this isn't your fault. Therefore don't feel obligated to do the fixing but it would be nice to have this argument named as `value_opt`.

Reply:
Ok changed all `value: Option<...` to `value_opt: Option<...` in the file.

Reply:
Thanks :pray: